### PR TITLE
feat(labware): update labware mapping

### LINKED ISSUE #3605: PD: finalize v1 labware model -> v3 loadName for all PD labware
See the Labware Definitions spreadsheet.

Closing this may require making new labware defs that we don't have yet.

## acceptance criteria

- Every labware available to users in production PD (1.1.0) has a mapping to a v3 labware definition
- All people working on labware confirm that the mapping is correct and final
- PD uses this labware mapping in migrating its protocols 1.1.0 -> 3.0.0

### PR DESCRIPTION
**Pull request by IanLondon**
## overview

Closes #3605. Derived from mapping in 3634

~Blocked until #3625 is merged, then this needs to be rebased on that for tests to pass~ This PR has been synced with edge and tests have been fixed

## changelog

* remove all generic_96_* labware
* update Python back_compat.py mappings

## review requests

- [ ] Make sure v1 -> v2 mappings are correct -- I'm aiming to make this PR finalize them for release
- [ ] Make sure `LW_NO_EQUIVALENT` is correct & exhaustive

### CODE REVIEW

File: labware-designer/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ To build a _regular_ labware, the `options` object should have the following sha
 | `version`    | [Version](#Version)       | no       | Version of the definition, should be incremented if non-metadata info is altered (defaults to `1`) |
 | `namespace`  | [Namepace](#Namespace)    | no       | Defaults to "custom_beta"                                                                          |
 
-This example generates [generic_96_wellplate_340ul_flat.json][]:
+This example generates [corning_96_wellplate_360ul_flat.json][]:
```
Comment (mcous) [inline labware-designer/README.md]: The actual contents of the object in the codeblock below will also need to be updated (that appears to be the case before as well). Could be punted to #3602

Comment (IanLondon) [reply labware-designer/README.md]: Added a note to that ticket. Good catch!

File: api/src/opentrons/protocol_api/back_compat.py [hunk 2]
```diff
@@ -194,45 +194,52 @@ class BCLabware:
...
+    LW_NO_EQUIVALENT = {'24-vial-rack', '48-vial-plate', '5ml-3x4',
+                        '96-well-plate-20mm', 'MALDI-plate',
+                        'T25-flask', 'T75-flask', 'biotix-resevoir',
+                        'e-gelgol', 'hampton-1ml-deep-block', 'point',
                         'rigaku-compact-crystallization-plate',
                         'small_vial_rack_16x45', 'temperature-plate',
-                        'tiprack-10ul-H', 'tiprack-200ul',
-                        'trough-12row-short', 'trough-1row-25ml',
-                        'trough-1row-test', 'tube-rack-.75ml',
+                        'tiprack-10ul-H', 'trough-12row-short',
```
[the remark this answers is not in the document]
Comment (sfoster1) [reply api/src/opentrons/protocol_api/back_compat.py]: Yes definitely

File: api/src/opentrons/protocol_api/back_compat.py [hunk 3]
```diff
@@ -195,44 +195,50 @@ def __init__(self, ctx: ProtocolContext) -> None:
...
 
     LW_NO_EQUIVALENT = {'24-vial-plate', '48-vial-plate', '5ml-3x4',
-                        '96-PCR-tall', '96-deep-well', '96-well-plate-20mm',
-                        'MALDI-plate', 'PCR-strip-tall', 'T25-flask',
-                        'T75-flask', 'alum-block-pcr-strips', 'e-gelgol',
-                        'hampton-1ml-deep-block', 'point',
-                        'opentrons-aluminum-block-PCR-strips-200ul',
+                        '96-well-plate-20mm', 'MALDI-plate',
+                        'T25-flask', 'T75-flask', 'alum-block-pcr-strips',
+                        'e-gelgol', 'hampton-1ml-deep-block', 'point',
```
[the remark this answers is not in the document]
Comment (mcous) [reply api/src/opentrons/protocol_api/back_compat.py]: My bad, will remove

File: api/src/opentrons/protocol_api/back_compat.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mcous) [reply api/src/opentrons/protocol_api/back_compat.py]: Update: `magdeck` and `tempdeck` were already special-cased and checked for. Not adding them to the LW_NO_EQUIVALENT map

https://github.com/Opentrons/opentrons/blob/6cac0b544c8c89316b9d9dea4f660e116493c730/api/src/opentrons/protocol_api/back_compat.py#L255-L261
